test/e2e/ibmcloud_test.go

```diff
@@ -98,10 +98,11 @@ func TestCreatePeerPodWithJob(t *testing.T) {
 }
 
 func TestCreatePeerPodAndCheckUserLogs(t *testing.T) {
-	assert := IBMCloudAssert{
-		vpc: pv.IBMCloudProps.VPC,
-	}
-	doTestCreatePeerPodAndCheckUserLogs(t, assert)
+	// assert := IBMCloudAssert{
+	// 	vpc: pv.IBMCloudProps.VPC,
+	// }
+	// doTestCreatePeerPodAndCheckUserLogs(t, assert)
+	t.Skip("Skipping Test until Kata issue is Fixed")
```

Comment:
Since if you mark a test as skipped after a test fails the test is still considered to have failed, we need to put the skip before the test that's expected to fail.

Since this test is known to fail in other cloud providers ([reference](https://github.com/confidential-containers/cloud-api-adaptor/pull/1236#issuecomment-1658927104) to the test failing when using the libvirt cloud provider) I think it would be good to move the skip to inside the `doTestCreatePeerPodAndCheckUserLogs` function.